src/layout/CustomButton/CustomButtonComponent.tsx

```diff
@@ -214,8 +214,25 @@ export const buttonStyles: { [style in CBTypes.ButtonStyle]: { color: ButtonColo
   secondary: { variant: 'secondary', color: 'first' },
 };
 
+function toShorthandSize(size?: CBTypes.ButtonSize): 'sm' | 'md' | 'lg' | undefined {
```

Comment:
Nitpick: Seems to me like this cannot really return `undefined`. Simply removing the return type definition here makes typescript infer the correct return type of `'sm' | 'md' | 'lg'`.